eng/init-poison.proj

```diff
@@ -21,6 +21,7 @@
     <ItemGroup>
       <PrebuiltPackages Include="$(PrebuiltPackagesPath)**/*.nupkg" />
       <PreviouslySourceBuiltPackages Include="$(PreviouslySourceBuiltPackagesPath)**/*.nupkg" />
+      <PreviouslySourceBuiltPackages Include="$(PreviouslySourceBuiltSharedComponentsPackagesPath)**/*.nupkg" />
```

Comment:
This doesn't seem right/complete.  Thoughts on reverting this now to include all required poison changes with https://github.com/dotnet/dotnet/issues/1112?

Reply:
Fixed in https://github.com/dotnet/dotnet/pull/1502/commits/8fdc989ada978f4cf69f429ea6ff062c5c38df6a